docs/core/setup/install-sdk.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Install .NET Core SDK on Windows, Linux, and macOS - .NET Core
+description: Learn how to install .NET Core on Windows, Linux, and macOS. Discover the dependencies required to develop .NET Core apps.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+ms.custom: "updateeachrelease"
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Install the .NET Core SDK
+
+In this article, you'll learn how to download and install the .NET Core SDK. The .NET Core SDK is used to create .NET Core apps and libraries. The .NET Core runtime is always installed with the SDK.
+
+You can download and install .NET Core directly with one of the following links:
+
+- [.NET Core 3.1 Preview 3 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.1)
```

Comment:
wouldn't this be install with an installer then?

Should you move there?

Reply:
I honestly just added it here for a shortcut of the "latest" preview. These download pages though go to the .NET website pages, which list all ways (including package managers) in a nice table matrix format. People get annoyed when they have to go down sections to find the right download link, I prefer to have a simple list at the start of the article to (when they're impatient) get them right to a download page.